jhove-core/src/main/java/edu/harvard/hul/ois/jhove/handler/JsonHandler.java

```diff
@@ -70,10 +70,33 @@
 	/** Handler release identifier. */
 	private static final String RELEASE = "1.0";
 
+	/** String release. */
+	private static final String RELEASE_CONSTANT = "release";
+
 	/** Handler release date. */
 	private static final int[] DATE = { 2019, 10, 18 };
 
-	/** Handler informative note. */
+	private static final String DATE_CONSTANT = "date";
```

Comment:
I think I'd choose `DATE_LABEL` rather than `DATE_CONSTANT`. Alternatively, you could change `private static final int[] DATE = { 2019, 10, 18 };` to `private static final int[] RELEASE_DATE = { 2019, 10, 18 };` to free up the `DATE` name.